src/CopyrighFixer/CopyrighFixer/config.h

```diff
@@ -0,0 +1,68 @@
+//#
+//# Copyright (C) 2021-2021 QuasarApp.
+//# Distributed under the lgplv3 software license, see the accompanying
+//# Everyone is permitted to copy and distribute verbatim copies
+//# of this license document, but changing it is not allowed.
+//#
+
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#include "CopyrighFixer_global.h"
+#include "CopyrighFixer/sign.h"
+
+
+namespace CopyrighFixer {
+
+/**
+ * @brief The Config class collects information about the current session
+ */
+class CopyrighFixer_EXPORT Config {
+public:
+    Config();
+
+    /**
+     * @brief setSourceDir This method set path to the source directory.
+     * @param pathToDir drink to the directory in which the files are to be signed. Example: /path/to/sources/dir
+     */
+    void setSourceDir(const QString &pathToDir);
+
+    /**
+     * @brief setSingValue This method sets a new value of the signatura.
+     * @param value Argument that passes the text of the signature.
```

Comment:
why do you tell about signature like about text?. Your signature it is separate c++ object not text.